manifests/centosimage.pp

```diff
@@ -29,18 +30,32 @@
     before  => Docker::Image['centosagent']
   }
 
+  # Mount the gem cache directory because docker doesn't follow symlinks
+  # mount --bind /var/cache/rubygems rubygems
+  file {'/var/docker/centosagent/rubygems':
+    ensure => directory,
+    before => Mount['/var/docker/centosagent/rubygems'],
+  }
+
+  mount {'/var/docker/centosagent/rubygems':
```

Comment:
@samuelson Installing the gems from `/var/cache/rubygems` is a serious PITA because you have to install deps in order. I think I'll create a shell function that does it so the student can just type something like `install_rspec_puppet` and `install_serverspec`. Thoughts?